vmm/src/memory_manager.rs

```diff
@@ -1810,9 +1810,17 @@ impl Aml for MemoryMethods {
                         &aml::Path::new("MINH"),
                         &aml::Path::new("LENH"),
                     ),
+                    &aml::If::new(
+                        &aml::LessThan::new(&aml::Path::new("MAXL"), &aml::Path::new("MINL")),
+                        vec![&aml::Add::new(
+                            &aml::Path::new("MAXH"),
+                            &aml::ONE,
+                            &aml::Path::new("MAXH"),
+                        )],
+                    ),
                     &aml::Subtract::new(
-                        &aml::Path::new("MAXH"),
-                        &aml::Path::new("MAXH"),
+                        &aml::Path::new("MAXL"),
+                        &aml::Path::new("MAXL"),
```

Comment:
Could you please explain what was wrong and the details of the calculation through the commit message?

Reply:
Yes, maybe you could show an ASL excerpt of the decompiled DSDT from within the guest.

Reply:
ACK, I need to figure out how to dump ASL there. What is usually used on Linux for this purpose?

Thanks

Reply:
`sudo acpidump  > acpi.out; acpixtract acpi.out; iasl dsdt.dat; cat dsdt.dsl`

Reply:
I think i can dump it with windbg.

Thanks.

Reply:
Ah, thanks. Or could dump it on Linux, too, might be easier :)

Thanks.

Reply:
Expanded on this for both the issue and ASL for the MCRS method in the commit message.

Thanks.